tests/zfs-tests/tests/functional/online_offline/online_offline_002_neg.ksh

```diff
@@ -30,6 +30,7 @@
 #
 
 . $STF_SUITE/include/libtest.shlib
+. $STF_SUITE/tests/functional/online_offline/online_offline.cfg
```

Comment:
Actually I may have misunderstood what you were saying.  We didn't create the config file it was already there.  We just made it be sourced because when when the config file is not source it only works for loop back devices.  We want this test to work for both physical disks and loopback devices.  But, yes we could purge these I believe: xport HOLES_FILESIZE=${HOLES_FILESIZE-"67108864"} # 64 Mb                      
export HOLES_BLKSIZE=${HOLES_BLKSIZE-"512"}                                     
export HOLES_SEED=${HOLES_SEED-""}                                              
export HOLES_FILEOFFSET=${HOLES_FILEOFFSET-""}                                  
export HOLES_COUNT=${HOLES_COUNT-"16384"}      # FILESIZE/BLKSIZE/8